build_v3.mac.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+dotnet build projects/GKv3/GEDKeeper3.sln -c MacOS_Debug -p:MacBuildBundle=true
+
+# For details how to create a proper DMG HSF+ file see here: https://stackoverflow.com/a/7553878/259946.
+# The ISO9660/HFS is a supported format for DMG files and can be treated as uncompressed DMG.
+
+# sudo apt install mkisofs
```

Comment:
`mkisofs`/`genisoimage` produces `ISO-9660` that is a valid DMG file that can be mounted on MacOS. See the link above how to make a compressed DMG file from ISO.